app/uploaders/image_uploader.rb

```diff
@@ -0,0 +1,65 @@
+# encoding: utf-8
+class ImageUploader < CarrierWave::Uploader::Base
+  include CarrierWave::MiniMagick
+
+  storage :file
```

Comment:
Please test that Imagemagick does not remove any EXIF information from the images. EXIF is very important because it provides information about who the author is, when the image was taken, what parameters the camera used etc.